org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/tests/unitTests.js

```diff
@@ -0,0 +1,48 @@
+var test = require('ava');
+var OpenT2T = require('opent2t').OpenT2T;
+var deviceState = require('./devicestate');
+var MockDevice = require('../../../test/mockDevice');
+var runThermostatTests = require('../../../test/thermostatTests');
+var translatorPath = require('path').join(__dirname, '..');
+var translator = undefined;
+
+function modifyDeviceState(deviceState, modifications) {
+    if(deviceState && modifications) {
+        for(var modification in modifications) {
+            deviceState.data.desired_state[modification] = modifications[modification];
+        }
+    }
+}
+
+function verifyPayload(payload, modification, t) {
+    t.deepEqual(payload, {desired_state: modification}, 'Verify payload');
+}
+
+var testData = {
```

Comment:
I combined this with the device state json file.

---
In reply to: [82066814](https://github.com/openT2T/translators/pull/106#discussion_r82066814) [](ancestors = 82066814)